### DISCUSSION TITLE
windows old system

### DISCUSSION THREAD

**Discussion by xiaokaidada** [r0x0r/pywebview#1247] [Q&A]
![GIF 2023-10-31 16-02-04](https://github.com/r0x0r/pywebview/assets/32946616/fe599d33-12c8-46e4-a5bd-17414417410c)

pywebview is not compatible with the old server, my system version is Windows Server 2012 R2, it will be automatically destroyed after running, do I need to install any environment

import webview
webview.create_window('Hello world', 'https://pywebview.flowrl.com/hello')
webview.start()

**Comment by r0x0r** [comment]
There is nothing to debug here without error log. Try running your exe from a command line.

**Reply by xiaokaidada** [reply]
Is it to run the exe with cmd?
